go/stellar/stellarsvc/service.go

```diff
@@ -220,6 +220,19 @@ func (s *Server) SendCLILocal(ctx context.Context, arg stellar1.SendCLILocalArg)
 	}, nil
 }
 
+func getPrimaryAccount(mctx libkb.MetaContext, remoter remote.Remoter) (acct stellar1.WalletAccountLocal, err error) {
```

Comment:
there is `stellar.GetOwnPrimaryAccountID()`...would that work instead of this?

Reply:
lol. i looked all over for this. i assumed it had to exist. yes. much better. thank you.